### DISCUSSION TITLE
In continuation of my previous discussion

### DISCUSSION THREAD

**Discussion by JustEp1c** [ant-design-blazor/ant-design-blazor#3713] [Q&A]
If I use `OnRow` parameter and set `row.Expanded` to true then my row isn't collapsing anymore

`Dictionary<string, object> OnRow(RowData<Data> row)
    {
        if (row.Data.Key == "1")
        {
            row.Expanded = true;
        }
        return new();
     }`

https://github.com/ant-design-blazor/ant-design-blazor/assets/85038502/d4bbbad3-e63e-4a1a-8122-40ab275f8382

**Comment by ElderJames** [comment]
Yes, this will make the row permanently expand (because Expanded is always true). To meet your needs, I think we need to add a property called DefaultExpandedRowKeys to specify the rows to expand during initialization
